connection_testcases.json

```diff
@@ -1401,153 +1435,98 @@
-    }
-  },
-  {
-    "env": {
-      "EDGEDB_DSN": "edgedb://?tls_verify_hostname=yes",
-      "EDGEDB_TLS_VERIFY_HOSTNAME": "false"
+      "tlsSecurity": "no_host_verification"
     },
     "result": {
       "address": ["localhost", 5656],
       "database": "edgedb",
       "user": "edgedb",
       "password": null,
       "tlsCAData": null,
-      "tlsVerifyHostname": false,
+      "tlsSecurity": "no_host_verification",
       "serverSettings": {}
     }
   },
   {
     "opts": {
       "dsn": "edgedb://",
-      "tlsCAFile": "/home/edgedb/test/cert.pem",
-      "tlsVerifyHostname": true
-    },
-    "fs": {
-      "files": {
-        "/home/edgedb/test/cert.pem": "-----BEGIN CERTIFICATE-----\nMIICEjCCAXsCAg36MA0GCSqGSIb3DQEBBQUAMIGbMQswCQYDVQQGEwJKUDEOMAwG\nA1UECBMFVG9reW8xEDAOBgNVBAcTB0NodW8ta3UxETAPBgNVBAoTCEZyYW5rNERE\nMRgwFgYDVQQLEw9XZWJDZXJ0IFN1cHBvcnQxGDAWBgNVBAMTD0ZyYW5rNEREIFdl\nYiBDQTEjMCEGCSqGSIb3DQEJARYUc3VwcG9ydEBmcmFuazRkZC5jb20wHhcNMTIw\nODIyMDUyNjU0WhcNMTcwODIxMDUyNjU0WjBKMQswCQYDVQQGEwJKUDEOMAwGA1UE\nCAwFVG9reW8xETAPBgNVBAoMCEZyYW5rNEREMRgwFgYDVQQDDA93d3cuZXhhbXBs\nZS5jb20wXDANBgkqhkiG9w0BAQEFAANLADBIAkEAm/xmkHmEQrurE/0re/jeFRLl\n8ZPjBop7uLHhnia7lQG/5zDtZIUC3RVpqDSwBuw/NTweGyuP+o8AG98HxqxTBwID\nAQABMA0GCSqGSIb3DQEBBQUAA4GBABS2TLuBeTPmcaTaUW/LCB2NYOy8GMdzR1mx\n8iBIu2H6/E2tiY3RIevV2OW61qY2/XRQg7YPxx3ffeUugX9F4J/iPnnu1zAxxyBy\n2VguKv4SWjRFoRkIfIlHX0qVviMhSlNy2ioFLy7JcPZb+v3ftDGywUqcBiVDoea0\nHn+GmxZA\n-----END CERTIFICATE-----\n"
-      }
+      "tlsSecurity": "insecure"
     },
     "result": {
       "address": ["localhost", 5656],
       "database": "edgedb",
       "user": "edgedb",
       "password": null,
-      "tlsCAData": "-----BEGIN CERTIFICATE-----\ncertdata\n-----END CERTIFICATE-----\n",
-      "tlsVerifyHostname": true,
+      "tlsCAData": null,
+      "tlsSecurity": "insecure",
       "serverSettings": {}
     }
   },
   {
-    "opts": {
-      "tlsCAFile": "/home/edgedb/test/cert.pem"
-    },
     "env": {
-      "EDGEDB_DSN": "edgedb://",
-      "EDGEDB_TLS_VERIFY_HOSTNAME": "true"
-    },
-    "fs": {
-      "files": {
-        "/home/edgedb/test/cert.pem": "-----BEGIN CERTIFICATE-----\nMIICEjCCAXsCAg36MA0GCSqGSIb3DQEBBQUAMIGbMQswCQYDVQQGEwJKUDEOMAwG\nA1UECBMFVG9reW8xEDAOBgNVBAcTB0NodW8ta3UxETAPBgNVBAoTCEZyYW5rNERE\nMRgwFgYDVQQLEw9XZWJDZXJ0IFN1cHBvcnQxGDAWBgNVBAMTD0ZyYW5rNEREIFdl\nYiBDQTEjMCEGCSqGSIb3DQEJARYUc3VwcG9ydEBmcmFuazRkZC5jb20wHhcNMTIw\nODIyMDUyNjU0WhcNMTcwODIxMDUyNjU0WjBKMQswCQYDVQQGEwJKUDEOMAwGA1UE\nCAwFVG9reW8xETAPBgNVBAoMCEZyYW5rNEREMRgwFgYDVQQDDA93d3cuZXhhbXBs\nZS5jb20wXDANBgkqhkiG9w0BAQEFAANLADBIAkEAm/xmkHmEQrurE/0re/jeFRLl\n8ZPjBop7uLHhnia7lQG/5zDtZIUC3RVpqDSwBuw/NTweGyuP+o8AG98HxqxTBwID\nAQABMA0GCSqGSIb3DQEBBQUAA4GBABS2TLuBeTPmcaTaUW/LCB2NYOy8GMdzR1mx\n8iBIu2H6/E2tiY3RIevV2OW61qY2/XRQg7YPxx3ffeUugX9F4J/iPnnu1zAxxyBy\n2VguKv4SWjRFoRkIfIlHX0qVviMhSlNy2ioFLy7JcPZb+v3ftDGywUqcBiVDoea0\nHn+GmxZA\n-----END CERTIFICATE-----\n"
-      }
+      "EDGEDB_DSN": "edgedb://?tls_verify_hostname=yes",
```

Comment:
This dsn is still using `tls_verify_hostname`

Reply:
I think this diff is out of date :confused: